core/esmf-aspect-model-document-generators/src/main/java/org/eclipse/esmf/aspectmodel/generator/zip/AspectModelZipGenerator.java

```diff
@@ -0,0 +1,49 @@
+package org.eclipse.esmf.aspectmodel.generator.zip;
+
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+import org.eclipse.esmf.aspectmodel.AspectModelFile;
+import org.eclipse.esmf.aspectmodel.serializer.AspectSerializer;
+import org.eclipse.esmf.metamodel.AspectModel;
+
+public class AspectModelZipGenerator {
```

Comment:
This should also talk about "zip" but about "namespace package". Maybe `AspectModelNamespacePackageCreator`?

Also I'd follow the pattern to make its usage obvious by implementing a functional interface and creating a static INSTANCE (like e.g. in `AspectSerializer`).

Proposal:
```suggestion
public class AspectModelNamespacePackageCreator implements BiConsumer<AspectModel, OutputStream> {

public static final AspectModelNamespacePackageCreator INSTANCE = new AspectModelNamespacePackageCreator();

private AspectModelNamespacePackageCreator() {
}

public void accept( final AspectModel aspectModel, OutputStream outputStream ) {
//...
}
```

(`accept()` instead of generateZipAspectModelArchive())